### PR TITLE
add changing task

### PR DESCRIPTION
**Pull request by Nikita-Koval**
I hope that was right.....

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
 import React, {useState, useEffect} from 'react';
 import './App.scss';
 import axios from 'axios';
+import accept from '../src/accept.ico'
```
Comment (usatovaDasha) [inline src/App.js]: missing ;

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
 import React, {useState, useEffect} from 'react';
```
Comment (usatovaDasha) [inline src/App.js]: imports style

File: src/App.js [hunk 3]
```diff
@@ -19,24 +21,57 @@ function App() {
...
+  const editTask = (index) => {
+    setIndex(index);
+    setNewText(tasks[index].text);
+  }; //edit task
+
+  const doneTask = async (index) => {
+    const { _id } = tasks[index];
+    if(newText === '') {
+      alert('Enter your text !') } else {
+    await axios.patch("http://localhost:8080/updateTask", {
```
Comment (usatovaDasha) [inline src/App.js]: please check tabs

File: src/App.js [hunk 4]
```diff
@@ -48,25 +83,65 @@ function App() {
...
       </header>
-      <div>
+
+      <div className='tasksCont'>
         {
-          tasks.map((task, index) => 
-            <div key={`task-${index}`}>
+          tasks.map((task, index) => {
+            return indexEdit === index ? (
+              <div className='textEdit' key={`task-${index}`}>
```
Comment (usatovaDasha) [inline src/App.js]: move to a component

File: src/App.js [hunk 5]
```diff
@@ -48,25 +83,65 @@ function App() {
...
+                src={accept}
+                onClick={() => doneTask(index)}
+              />
+                <img
+                src={canc}
+                onClick={() => cancelTask(index)}
+              />
+              </div>
+            ) : (
+            <div
```
Comment (usatovaDasha) [inline src/App.js]: move to a component

File: src/App.scss [hunk 6]
```diff
@@ -1,50 +1,160 @@
...
+  box-shadow: inset 0 0 15px rgba(0, 0, 0, .5);
+}
+
+.wrapper {
+  background-color: rgb(250, 43, 43);
+}
+
+.wrapper:nth-child(2) {
+  background-color: #e98636c7;
+}
```
Comment (usatovaDasha) [inline src/App.scss:93]: add empty lines

File: src/App.js (hunk 5 above)
Comment (Nikita-Koval) [reply src/App.js]: Done

File: src/App.js (hunk 4 above)
Comment (Nikita-Koval) [reply src/App.js]: Done

File: src/App.js (hunk 3 above)
Comment (Nikita-Koval) [reply src/App.js]: Done

File: src/App.scss (hunk 6 above)
Comment (Nikita-Koval) [reply src/App.scss:93]: Done

File: src/App.js (hunk 2 above)
Comment (Nikita-Koval) [reply src/App.js]: Done

File: src/App.js (hunk 1 above)
Comment (Nikita-Koval) [reply src/App.js]: Done

**Pull request merged**